src/tools/genpolicy/rules.rego

```diff
@@ -60,7 +60,9 @@ CreateContainerRequest {
     i_storages := input.storages
     i_devices := input.devices
 
-    # Check if any element from the policy_data.containers array allows the input request.
+    print("CreateContainerRequest: i_oci.Hooks =", i_oci.Hooks)
+    is_null(i_oci.Hooks)
```

Comment:
@Redent0r , we already have this rule in allow_create_container_input() nowadays.